root/search/components/EventResults.js

```diff
@@ -28,8 +28,9 @@ function buildResult(result, index) {
   return (
     <tr className={loopParity(index)} data-score={score} key={event.id}>
       <td>
-        <DescriptiveLink entity={event} />
+        <a href={'/event/'+event.id}>{event.name}</a>
       </td>
+      <td>{formatDatePeriod(event)}{event.time ? ", "+event.time : null}</td>
```

Comment:
The comma separator is not localizable. The string concatenation doesn’t support right-to-left locales either. I suggest adding a function instead, following the example of `i18n.addColon`.

Reply:
I like the columns separate, personally, since it's easier for me to read.

Reply:
So should I put them together or keep them seperate?

Reply:
Merging the columns seems unrelated to the ticket, so I'd probably leave it out of the PR unless there's a better rationale ("they were on the other side of the screen" doesn't make sense to me since you can move them without merging them, and "taking too much space" doesn't appear to be true to me).

There are some other places we show time in a separate column so it introduces another inconsistency is all.

Reply:
Okay, no problem--I was just acting on the comments on the Jira ticket :)

Reply:
Thank you for having tried at least!

It was suggested as _optional_ because it might be a hassle for little gain. In the case of dates range, it would be more clear to have start time just after start date rather than after end date.